data/json/items/melee/bludgeons.json

```
@@ -814,9 +813,9 @@
     "symbol": "/",
     "material": [ "wood", "steel" ],
     "looks_like": "hammer_sledge",
-    "flags": [ "NONCONDUCTIVE", "ALWAYS_TWOHAND" ],
+    "flags": [ "DURABLE_MELEE", "NONCONDUCTIVE", "ALWAYS_TWOHAND" ],
```

```suggestion
    "flags": [  "NONCONDUCTIVE", "ALWAYS_TWOHAND" ],
```
Sledges aren’t durable in melee. I’ve thrown too many heads. Unless this is a one piece cast metal sledge but then it loses non-conductive

---

I think bashing solid objects wears a tool down much more than hitting relatively soft targets. If I went to town on a bunch of concrete slabs with a mace I think it'd wear down in a similar time